app.js

```diff
@@ -61,31 +62,39 @@ export default class App extends Component {
   }
 
   clickGenreHandler(event) {
-    let state = Object.assign({}, this.state);
-    state.oArgs.category = event.target.textContent.toLowerCase();
-    state.oArgs.page_number = 1;
-    state.oArgs.keywords = '';
-    this.setState({
+    let newOargs = Object.assign({}, this.state.oArgs, {
```

Comment:
Only use `let` if you plan to reassign the variable with `=`. Favor `const` where possible.